paasta_tools/kubernetes_tools.py

```diff
@@ -1105,6 +1106,14 @@ def get_kubernetes_containers(
         )
         return containers
 
+    def get_readiness_probe(
+        self, service_namespace_config: ServiceNamespaceConfig
+    ) -> Optional[V1Probe]:
+        if service_namespace_config.is_in_smartstack():
+            return None
```

Comment:
aside from causing a big bounce, how do we feel about always returning the liveness probe? I think this would most closely match setup_marathon_job's behavior, which considered both the service's healthcheck (service's liveness check) and whether it's in smartstack (hacheck container's readiness check).

Reply:
I think I am roughly down with that.  That would mean that each service would need to respond to healthcheck 13 times a minute -- every 10s for both the liveness and readiness probes plus once per minutes from hacheck.  Do you think that is ok?

Reply:
I think our hacheck is configured to [cache for only 1 second](https://github.com/Yelp/paasta/blob/7eba1996de7a51a016536e2ee4b2f3c9de413c09/yelp_package/dockerfiles/hacheck-sidecar/hacheck.conf.yaml#L1).